### ISSUE TITLE
eav step error. although proper <rename> present. 

### ISSUE THREAD

**Issue by dmcmillin** [magento/data-migration-tool#101]
following error occurs when running migration. 

```
100% [============================] Remaining Time: 1 sec
[2016-06-24 21:29:32][INFO][mode: data][stage: volume check][step: EAV Step]: started
100% [============================] Remaining Time: 1 sec
[2016-06-24 21:29:32][ERROR]: Incorrect value: boxmenu/boxmenu in: eav_attribute.source_model for attribute_code=ship_box
[2016-06-24 21:29:32][INFO][mode: data][stage: volume check][step: EAV Step]: Error occurred. Rollback.
[2016-06-24 21:29:32][INFO][mode: data][stage: volume check][step: EAV Step]: EAV Step: rollback
[2016-06-24 21:29:34][INFO][mode: data][stage: volume check][step: EAV Step]: Please fix errors and run Migration Tool again

```

added to class-map.xml.dist 

<rename>
	<from>boxmenu/boxmenu</from>
	<to />
</rename>

still displays error. Please assist.

**Comment by victor-v-rad** [comment]
Hi
Adding attribute ship_box into eav-attribute-groups.xml should solve you problem

**Comment by dmcmillin** [comment]
I did already add that to eav-attribute-groups.xml still erroring. Thanks!!!

This is what my eav-attribute-groups.xml looks like

```
<groups xmlns:xs="http://www.w3.org/2001/XMLSchema-instance" xs:noNamespaceSchemaLocation="../groups.xsd">
    <group name="ignore">
        <attribute>msrp_enabled</attribute>
        <attribute>group_price</attribute>
        <attribute>ship_box</attribute>
    </group>
</groups>

```

**Comment by victor-v-rad** [comment]
Make sure you have this modified eav-attribute-groups.xml in your config.xml
The thing is that in config.xml you have eav-attribute-groups.xml.dist by default

**Comment by dmcmillin** [comment]
I had to refresh / clean cache and try again server side. Now its spitting out a different area on frontend after finalizing the migration. 

```
Warning: preg_match(): Delimiter must not be alphanumeric or backslash in /home/vasaripl/public_html/magento2/lib/internal/Magento/Framework/View/DesignExceptions.php on line 70
#0 [internal function]: Magento\Framework\App\ErrorHandler->handler(2, 'preg_match(): D...', '/home/vasaripl/...', 70, Array)
#1 /home/vasaripl/public_html/magento2/lib/internal/Magento/Framework/View/DesignExceptions.php(70): preg_match('Googlebot-Mobil...', 'Mozilla/5.0 (Wi...')
#2 /home/vasaripl/public_html/magento2/app/code/Magento/PageCache/Model/App/CacheIdentifierPlugin.php(43): Magento\Framework\View\DesignExceptions->getThemeByRequest(Object(Magento\Framework\App\Request\Http))
#3 /home/vasaripl/public_html/magento2/lib/internal/Magento/Framework/Interception/Interceptor.php(150): Magento\PageCache\Model\App\CacheIdentifierPlugin->afterGetValue(Object(Magento\Framework\App\PageCache\Identifier\Interceptor), '92abe4ddb8e2b9c...')
#4 /home/vasaripl/public_html/magento2/var/generation/Magento/Framework/App/PageCache/Identifier/Interceptor.php(26): Magento\Framework\App\PageCache\Identifier\Interceptor->___callPlugins('getValue', Array, Array)
#5 /home/vasaripl/public_html/magento2/lib/internal/Magento/Framework/App/PageCache/Kernel.php(51): Magento\Framework\App\PageCache\Identifier\Interceptor->getValue()
#6 /home/vasaripl/public_html/magento2/app/code/Magento/PageCache/Model/App/FrontController/BuiltinPlugin.php(71): Magento\Framework\App\PageCache\Kernel->load()
#7 [internal function]: Magento\PageCache\Model\App\FrontController\BuiltinPlugin->aroundDispatch(Object(Magento\Framework\App\FrontController\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#8 /home/vasaripl/public_html/magento2/lib/internal/Magento/Framework/Interception/Chain/Chain.php(68): call_user_func_array(Array, Array)
#9 /home/vasaripl/public_html/magento2/lib/internal/Magento/Framework/Interception/Interceptor.php(136): Magento\Framework\Interception\Chain\Chain->invokeNext('Magento\\Framewo...', 'dispatch', Object(Magento\Framework\App\FrontController\Interceptor), Array, 'front-controlle...')
#10 /home/vasaripl/public_html/magento2/app/code/Magento/PageCache/Model/App/FrontController/VarnishPlugin.php(55): Magento\Framework\App\FrontController\Interceptor->Magento\Framework\Interception\{closure}(Object(Magento\Framework\App\Request\Http))
#11 [internal function]: Magento\PageCache\Model\App\FrontController\VarnishPlugin->aroundDispatch(Object(Magento\Framework\App\FrontController\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#12 /home/vasaripl/public_html/magento2/lib/internal/Magento/Framework/Interception/Interceptor.php(141): call_user_func_array(Array, Array)
#13 /home/vasaripl/public_html/magento2/var/generation/Magento/Framework/App/FrontController/Interceptor.php(26): Magento\Framework\App\FrontController\Interceptor->___callPlugins('dispatch', Array, Array)
#14 /home/vasaripl/public_html/magento2/lib/internal/Magento/Framework/App/Http.php(115): Magento\Framework\App\FrontController\Interceptor->dispatch(Object(Magento\Framework\App\Request\Http))
#15 /home/vasaripl/public_html/magento2/lib/internal/Magento/Framework/App/Bootstrap.php(258): Magento\Framework\App\Http->launch()
#16 /home/vasaripl/public_html/magento2/index.php(39): Magento\Framework\App\Bootstrap->run(Object(Magento\Framework\App\Http))
#17 {main}
```

**Comment by victor-v-rad** [comment]
The latest issue 
{quote}
Warning: preg_match(): Delimiter must not be alphanumeric or backslash
{quote}
you faced is different then in the original post. Could you please create separate issue for this

**Comment by victor-v-rad** [comment]
An internal issue MAGETWO-55275 was created to address the case with Design Exceptions

**Comment by onepack** [comment]
@victor-v-rad we face the same issue after a successful data import as @dmcmillin.  
Next to the frontend error we face issues in the backend under content -> design configuration.
All themes values have become empty and It is not longer possible to set a theme as loading the edit page will not work. -> Load for ever.
I found that the database design_config_grid_flat table had the name of the M1 theme after the data import. In M2 this needs to be a numerical value and so I changed it to the default 1 (blank or 2 luma). (Also the number of the categories source needs to be set in core_config_data.)
After the database change (string to numeric value) I was able to set the theme in the backend for one time only and after that the never ending loading occurred again.
The frontend error continues even after all these steps and clearing the cache numerous times.
